cpp/include/ray/api.h

```diff
@@ -39,6 +39,14 @@ class Ray {
   template <typename T>
   static ObjectRef<T> Put(const T &obj);
 
+  /// Get a single object from the object store.
+  /// This method will be blocked until all the objects are ready.
+  ///
+  /// \param[in] object The object reference which should be got.
+  /// \return shared pointer of the result.
+  template <typename T>
+  static std::shared_ptr<T> Get(const ObjectRef<T> &object);
```

Comment:
We can also remove `Get(const std::vector<ObjectID> &ids)` now

Reply:
We don't support get object for ID?

Reply:
Right. Actually we don't need to expose ObjectID to users at all.

Reply:
ok